blog/2024/2024-03-15-bestpractices-emailsupport.md

```diff
@@ -0,0 +1,54 @@
+---
+slug: Best Practices Series - Email Support
+title: Best Practices Series - Email Support
+authors:
+  - name: Vikas Gupta
+    title: Director of Integration Architecture & Partner Support
+tags: ['bestpractices', 'Integrations', 'Partners', 'Developers', 'Community']
+hide_table_of_contents: false
+---
+
+# Best Practices Series - Email Support
```

Comment:
Please do not set h1s, add to the front matter a "tittle" attribute and put your title in